Portfolio Template 2

sunga12 opened the pull request:
Added the following elements to the Navigation bar with typography

- Name
- Portfolio
- About
- Contact
- Email text (Not icon)
- Page Background colour

sunga12 committed:
Add Navbar section & Page background colour

sunga12 committed:
Fix Stylelint Errors

sunga12 committed:
Add Main Headline text & typography

sunga12 committed:
Fix Stylelint Errors

sunga12 committed:
Add Aside and Email Icons

sunga12 committed:
Add Aside & Email icons. Fix Stylelint Errors

sunga12 committed:
Update README file for current project

sunga12 committed:
Create LICENSE

This is a license for the PORTFOLIO project

sunga12 commented:
For this project, the requirements were to:

- Use the [HTML&CSS linter]

The 2 sections to build for this project are:

- The toolbar (or header):
- The headline section (right after the header):

Requirements completed:

- Used the [HTML&CSS linter]
- Built the toolbar
- Built the headline section

Favourezeugwa commented on index.html:
```diff
@@ -5,8 +5,48 @@
     
+<header>
+    <nav>
+        <div class="name">
+            <div class="name-text">Sungabanja Thawethe</div>
+        </div>
+        <div class="item-navigation"> 
+            <div class="frame-port">
+                <div class="portfolio">Portfolio</div>
+            </div>
+            <div class="frame-about">
+                <div class="about">About</div>
+            </div>
+            <div class="frame-contact">
+                <div class="contact">Contact</div>
+            </div>       
+        </div>
+        <div class="email-box">
+            <img class="icons" src="Union.png" alt="Github icon">
+            
+        </div>
```
- For the scope of this project, you are expected to build the mobile version of your portfolio first. Can you kindly replace the highlighted lines of code with just the hamburger menu alone? By next week, you will start working on the desktop version, then you can implement this. 🤗 

Here is what I mean 👇🏽 
| Your Toolbar  |Figma Toolbar |
| :---:   | :---: | 
| <img width="382" alt="Screenshot 2023-01-12 at 18 46 50" src="https://user-images.githubusercontent.com/99246359/212211592-4963501d-4f5c-4e71-bf39-998e7490155e.png"> |<img width="388" alt="Screenshot 2023-01-12 at 18 47 09" src="https://user-images.githubusercontent.com/99246359/212211599-9ef1dda6-b89c-4139-b879-cd951b1f58e8.png">  |

sunga12 replied:
Hi @Favourezeugwa ,

I was unable to find the mobile version of the page. Thank you for the screenshots. How do I find the mobile version of the template?

Favourezeugwa commented on index.html:
```diff
@@ -5,8 +5,48 @@
+        <div class="SD">I'm a Software Developer</div>
+        <div class="info">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</div>
+    </div>
+</main>
```
- Instead of a `main` tag, can we kindly replace the container of this section with a `section` tag.  It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

- Can we ensure to stick to the design on Figma as much as possible? Currently, you have your social icons vertially aligned on the left, which is not right. 

- Also, can we kindly ensure to use the right font size and line height for our page's main title `Hey There. I'm Sunga` and paragraph `I can help you build a product . . . .`? What you have currently are a bit too large. 

Here is what I mean 👇🏽 

| Your Design  | Figma Design |
| :---:   | :---: | 
| <img width="387" alt="Screenshot 2023-01-12 at 18 56 01" src="https://user-images.githubusercontent.com/99246359/212212454-1505949a-76d1-4725-ae31-0a4739af229b.png"> |<img width="391" alt="Screenshot 2023-01-12 at 18 33 23" src="https://user-images.githubusercontent.com/99246359/212212489-c3e8bf7d-37ae-4c27-ae0b-29ce470fc59b.png">  |

Favourezeugwa commented on index.html:
```diff
@@ -5,8 +5,48 @@
+    </aside>
+
+    <div class="main-text">
+        <div class="hello-me">Hey There. I'm Sunga</div>
+        <div class="SD">I'm a Software Developer</div>
+        <div class="info">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</div>
```
- I think it will be better to use `<hx>` and `<p>` tags to represent text instead of divs alone. Why don't we replace the divs to `<h1>` and `<p>` in the cases that apply?  

For example, line 47 should be a paragraph and in a `<p>` tag. 🤗

Favourezeugwa reviewed (changes requested):
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @sunga12  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![so-close-brianna](https://user-images.githubusercontent.com/99246359/201470450-173d4ea7-fe78-470e-a872-142a58b56294.gif)


## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sunga12 committed:
Update Headline based on Review Request 1

sunga12 committed:
Set Margins on Section elements to 0

sunga12 commented:
Requirements:

- Change div tags to appropriate h1, h3 & p tags:
- Replace the container of headline section with a section tag
- Place social media Icons below section text as is in mobile template
- Update typography to be the same as the template
- Replace Item navigation with Hambuger menu

Requirements completed:

- Changed div tags to appropriate h1, h3 & p tags:
- Replaced the container of headline section with a section tag
- Placed social media Icons below section text as is in mobile template
- Updated typography to be the same as the template

sunga12 committed:
Update Navigation Bar based on Review

sunga12 committed:
Add Illustration & Fix section oversize problem

sunga12 commented:
Requirements:

- Replace Item navigation with Hamburger menu
- Add illustration to page

Requirements completed:

- Replaced Item navigation with Hamburger menu
- Added illustration to page

Other work done:

- Updated Header and body to solve the problem which caused the section to overflow beyond the Navigation bar when the screen width was increased.

franclobo commented on index.html:
```diff
@@ -5,8 +5,28 @@
+        <h1 class="hello-me">Hey There. I'm Sunga</h1>
+        <h3 class="SD">I'm a Software Developer</h3>
+        <p class="info">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <img class="icons" src="Social Media.png" alt="Social Media icons">
```
- **[OPTIONAL]** The last reviewer didn't ask you but this is a requirement: The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`. Please 🙏🏽 list the social media icons using `ul` tag for good practices of HTML&CSS.

franclobo reviewed (approved):
Hi @sunga12, 

# STATUS: APPROVED 😃 

![gif-good-job-79](https://user-images.githubusercontent.com/58642949/212320614-49dab297-8482-4e21-8e2b-17f7c38e78dc.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️
- Professional README file ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
